File: src/main/java/facts/stmt/Assign/RegisterAssign.java
package facts.stmt.Assign;

import astnodes.exp.Expr;
import astnodes.exp.Var;

/**
 * Load fact
 */
public class RegisterAssign extends Assign {
    public RegisterAssign(String pc, Var lhsExp, Expr expr ) {
        super(pc, lhsExp, expr);
    }
}

File: src/main/java/facts/stmt/Assign/RegisterAssign.scala
package facts.stmt.Assign

import astnodes.exp.Expr
import astnodes.exp.Var
import facts.stmt.Stmt

// TODO check if we need override pc
/** Load fact
  */
case class RegisterAssign(override val pc: String, val lhsExp: Var, val expr: Expr) extends Assign(pc, lhsExp, expr) {}
